Assets/Stack mini game/Scripts/GameplayManager.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class GameplayManager : MonoBehaviour
+{
+    [SerializeField] private CubeSpawner cubeSpawner;
+
+    [SerializeField] private CubeStack cubeStack;
+
+    [SerializeField] private int fullWinScore;
```

Comment:
These would be good to have in a scriptable object

Reply:
Are you sure? CubeStack is a monobehaviour on a gameobject. It is not just some data. For example it has a variable named  CubeCornersPositionPile. From what I understand about scriptable object, you can't have variables that will be modified while the game is being played because the values are not reset when you restart the game.